code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -2090,7 +2090,7 @@ shaved
 
 	//Empty list is unrestricted. Should only restrict the ones that make NO SENSE on other species,
 	//like Tajaran inner-ear coloring overlay stuff.
-	species_allowed = list()
+	species_allowed = list(SPECIES_HUMAN,SPECIES_SKRELL,SPECIES_TAJ,SPECIES_HUMAN_VATBORN,SPECIES_UNATHI,SPECIES_ZADDAT)

```

Comment:
Uh. No. We don't want markings as a whole restricted to just those. That's Polaris thing.

Reply:
Good catch because I didn't notice that at first, fix is up